components/03-organisms/site-footer/site-footer.twig

```diff
@@ -20,61 +20,60 @@
 } %}
 
 <footer {{ add_attributes(site_footer__attributes) }}>
-  <div {{ bem('inner', [], site_footer__base_class) }}>
-    {# Primary #}
-    <div {{ bem('primary', [], site_footer__base_class) }}>
-      {# Branding #}
-      <div {{ bem('branding', [], site_footer__base_class) }}>
-        {% block site_footer__logo %}
-          {% include "@page-layouts/placeholder/placeholder.twig" with {
-            placeholder: 'Logo',
-            placeholder__type: 'element',
-          } %}
-        {% endblock %}
-        {% block site_footer__text %}
-          {% include "@page-layouts/placeholder/placeholder.twig" with {
+	<div
```

Comment:
It looks like your editor also converted all of our spaces to tabs. I think that's a default setting, and we might have reconfigured your VSCode to use spaces by default - after you had saved this one. There's a way to quickly "convert tabs to spaces" in VSCode. Let's zoom to make sure that gets done for you so that I can actually review this. It's hard to see what your actual changes are... if there even are any.